Minio/MinioClient.cs

```diff
@@ -87,13 +87,18 @@ public MinioClient()
         Region = "";
         SessionToken = "";
         Provider = null;
-        HTTPClient = new HttpClient();
+        var clientHandler = new HttpClientHandler();
+        clientHandler.ServerCertificateCustomValidationCallback =
+            (sender, cert, chain, sslPolicyErrors) => { return true; };
```

Comment:
if you do this then SSL verification will be turned-off in the SDK we cannot have an insecure TLS enforcing HTTP client, we need to make sure by default we ask for trust and security. 

I am not why this code is needed.

Reply:
I'll remove it.

Reply:
Removed